src/models/logic/utils/workerpool.ts

```diff
@@ -31,7 +31,9 @@ const getSignersFromPubKeyAndRegularKeyOwner = (
             return regularKeyOwner;
         }
 
-        return PlatformAddress.fromPublic(signerPubKey, networkId);
+        return PlatformAddress.fromPublic(signerPubKey, {
+            networkId
+        });
```

Comment:
Oops, `toString()` is required since it should return `string` not `PlatformAddress`.